README.md

```diff
@@ -7,42 +7,52 @@
 
 ## Overview
 
-This add-on integrates Tsh into your [DDEV](https://ddev.com/) project.
+This add-on integrates [Teleport](https://goteleport.com/) and its command line tool, [tsh](https://goteleport.com/docs/connect-your-client/tsh/) into your [DDEV](https://ddev.com/) project.
 
 ## Installation
 
+It is recommended to use this add-on in conjunction with the [DDEV Kubernetes add-on](https://github.com/MurzNN/ddev-kubernetes).
+
+```shell
+ ddev add-on get MurzNN/ddev-kubernetes
+```
+
+During installation, you will be asked to input your Teleport username, proxy, cluster, and k8s cluster (if applicable). You can skip any if you don't need them or don't know them yet.
+
 ```bash
 ddev add-on get froboy/ddev-tsh
 ddev restart
 ```
 
-After installation, make sure to commit the `.ddev` directory to version control.
+After installation, adjust any files for your project needs.
```

Comment:
In general, add-on files that have #ddev-generated in them can be replaced by this add-on in an upgrade, and those that don't won't be.

Reply:
```suggestion
After installation, adjust any files for your project needs. Remove `#ddev-generated` from any files that you edit, as they will be overwritten on the next `ddev restart`.
```